xml/art-obs-beginners-guide.xml

```diff
@@ -558,26 +555,32 @@ Cmnd_Alias  OSC_CMD = /usr/bin/osc, /usr/bin/build
   <title>Creating a New Project</title>
   <para>
    This section covers how to create packages from a simple C++ project
-   hosted on &gh; (the <quote>upstream project</quote>). We assume, this
+   hosted on &gh; (the <quote>upstream project</quote>). We assume that this
    project contains source code which you want to package for different
-   &suse; distributions. Furthermore, &osccmd; is already installed on your
-   machine.
+   &suse; distributions.
   </para>
   <para>
    To create a package from the upstream project, do the following:
   </para>
   <procedure xml:id="pro.obsbg.uc.basicprj">
    <step>
+    <remark>
+     Again, this is very specific to the openSUSE Build Service! Also, iirc
+     refusal to build is not a thing. OBS itself is totally agnostic to that,
+     but when you submit something to openSUSE (so again openSUSE BS-specific),
+     the license will be checked by some totally optional extra web service.
+     - sknorr, 2017-08-16
```

Comment:
right, we use the "opensuse"  and "novell" profile already in our default DC files.